src/xmlnode.cpp

```diff
@@ -157,10 +157,13 @@ XmlNodePtr XmlNode::getParent()
     return parentHandle;
 }
 
-std::string XmlNode::convertToString() {
+std::string XmlNode::convertToString(int format) {
```

Comment:
I would have preferred an enumeration here.

Reply:
I thought I would mimic what `xmlNodeDump()` does, but if anything I would actually use a `bool`.

Reply:
I would use `enum` if there was going to be more than two formats otherwise `bool` if there were only going to be two.  Either way not `int`.

Reply:
For whatever reason, libxml2 uses an `int`, but there is really only two values: `0` and `1` for no formatting and formatting, respectively.

Ok, I am going to use a `bool`.

Reply:
libxml2 is a C API which has no native bool type....